content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -66,48 +53,90 @@ When constructing your funnel, you can filter globally or on a step.
 
-  **Note**: Only completed RUM sessions contribute to conversion rates.
+- **Conversion count**: A count of users who went through the funnel you've defined.
+- **Conversion rate**: A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+- **Time to convert**: The time it took for the user to complete the step events. This is not available for the steps visualization, it changes to timeseries when selected.
 
-- **Time to convert** - The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the [**filter**](#filtering) selector to scope to the various criteria you want to define. These filters are applied to all steps in your funnel.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## Conversion computing metrics
 
-## Changing the visualization
+### How Datadog computes conversion metrics
+Consider a funnel with events `A → B → C` and event steps `A, A, A, B, C, C`. 
+
+
+In this case, Datadog counts one conversion. This is because the conversion calculation matches only the first occurrence of event **A** and the first occurrence of event **C** in the sequence. 
+
+
+To further illustrate, if the user performs the event sequence `A, A, A, B, C, C, A, B, C`, Datadog counts two conversions. The first conversion completes with the sequence `A, A, A, B, C, C`, and the second conversion begins with the next occurrence of `A`.
 
-{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_change_viz.mp4" alt="Click the visualization dropdown to select a different view" video=true >}}
 
+Datadog calculates the average time between steps based on the average duration between the first and last step of each conversion.
+
+If you analyze your funnel by **user** or by **Account**, you can define your conversion time frame in hours or days of the first event. The default time frame for conversions is one day (a 24-hour window, not a calendar date) to determine if a conversion happened or not.
+
+<div class="alert alert-info"> Any action or view that happens between two steps in a funnel does not impact the step-by-step or overall conversion rate. As long as step 1 and step 2 happen in the right order in a given session at least once, it counts as a single converted session.</div>
+
+### Choose a conversion counting method
+
+When computing your conversion, you can choose between two counting methods: **Unique** or **Total**. 
```

Comment:
Notice that here, you've capitalized your options, but you haven't earlier in the page (for example, "session, user, or account"). I would pick one way of doing it and stay consistent! It doesn't really matter which you choose, but as you self-edit, this is a great type of inconsistency to keep an eye out for.